LN-Zap/zap-desktop
# refactor: extract form styles to theme

## Description:

Extracts form input styles to the theme and refactors form components to use new `InputField` component in order to standardize sub component layout

## Motivation and Context:

Ability to style something as an Input field outside of the context of Informed.

## How Has This Been Tested?

- Verify forms look correct in the app
- Verify InputField component in the storybook

## Types of changes:

Refactor

## Checklist:
